godot/src/lib.rs

```diff
@@ -131,6 +131,9 @@ compile_error!("Must opt-in using `experimental-wasm` Cargo feature; keep in min
 #[cfg(all(feature = "double-precision", not(feature = "api-custom")))]
 compile_error!("The feature `double-precision` currently requires `api-custom` due to incompatibilities in the GDExtension API JSON.");
 
+#[cfg(all(feature = "user-docgen", not(feature = "api-custom")))]
+compile_error!("Documentation generation requires 4.3 (`api-custom`).");
```

Comment:
This check is incorrect, it should check for `since_api = "4.3"` or `before_api = "4.3"`.

`api_custom` doesn't give information about the API level.

Reply:
I cant do that in `godot`.

Reply:
why not

Reply:
because theres no `build.rs` in godot and `since_api` isnt defined

Reply:
That's no reason to use an incorrect check though. You can have the assertion in another crate, too.